Client/Frontend/Browser/Tabs/TabTrayViewController.swift

```diff
@@ -369,3 +362,49 @@ extension TabTrayViewController: RemotePanelDelegate {
             presentThemedViewController(navItemLocation: .Left, navItemText: .Close, vcBeingPresented: controller, topTabsVisible: UIDevice.current.userInterfaceIdiom == .pad)
         }
 }
+
+// MARK: - Handle Notifications
```

Comment:
A pattern we've been following has been that the `Themable` extension only has themable-related stuff. We should not also handle notifications and update button titles here. Those both seem to better be a part of the main class. If we had more notification stuff here, I'd argue then it should have its own extension, but just being one function, I don't know if that's needed.